perun-core/src/main/java/cz/metacentrum/perun/core/api/ResourcesManager.java

```diff
@@ -65,6 +65,21 @@
 	 */
 	Resource createResource(PerunSession perunSession, Resource resource, Vo vo, Facility facility) throws InternalErrorException, PrivilegeException, VoNotExistsException, FacilityNotExistsException, ResourceExistsException;
 
+	/**
+	 * Copy "template" settings(attributes,services,tags (if exists),...) from user's another existing resource and create new resource with this template.
+	 * Resource can be from any of user's facilities.
+	 *
+	 * @param perunSession
+	 * @param templateResource template resource to copy
+	 * @param destinationResource destination resource containing destination facility, VO & resource name.
```

Comment:
It looks like & is used for something different in javadoc. Use "and" instead please.